src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -24,13 +24,49 @@ public static WinRmTool connect(String address, String username, String password
         return tool;
     }
 
+    /**
+     * Execute a list of Windows Native commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"\r\n"</code> delimiter and a terminating one.
+     * It is creating a new Shell on the destination host each time it is being called.
+     * @param commands
+     * @deprecated use the {@link #executeCommand} instead and transform your commands list explicitly
+     */
+    @Deprecated
     public WinRmToolResponse executeScript(List<String> commands) {
         Response response = session.run_cmd(compileScript(commands));
         return responseToWinRmToolResponse(response);
     }
 
-    public WinRmToolResponse executePs(List<String> commands) {
-        Response response = session.run_ps(compileScript(commands));
+    /**
+     * Executes a Native Windows command.
+     * It is creating a new Shell on the destination host each time it is being called.
+     * @param command
+     */
+    public WinRmToolResponse executeCommand(String command) {
+        Response response = session.run_cmd(command);
+        return responseToWinRmToolResponse(response);
+    }
+
+    /**
+     * Execute a list of Power Shell commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"\r\n"</code> delimiter and a terminating one.
+     * It is creating a new Shell on the destination host each time it is being called
```

Comment:
As above (for implementation detail in the javadoc, and for saying when it was deprecated, and fixing the `@link`